internal/ingress/annotations/parser/main_test.go

```diff
@@ -34,6 +37,32 @@ func buildIngress() *extensions.Ingress {
 	}
 }
 
+func buildPlugin() *pluginv1.KongPlugin {
```

Comment:
Same comment here, do these tests need to be repeated? The don't seem to be testing anything specific to a plugin, consumer or ingress but are operating on the ObjectMeta struct.